onnxmltools/convert/sklearn/operator_converters/KNN.py

```diff
@@ -5,10 +5,18 @@
 # --------------------------------------------------------------------------
 
 from ....proto import onnx_proto
-from ...common._apply_operation import apply_abs, apply_mul, apply_reshape, apply_sub
+from ...common._apply_operation import apply_abs, apply_mul, apply_reshape, apply_sub, apply_cast
 from ...common._registration import register_converter
+from .OneHotEncoder import convert_sklearn_one_hot_encoder
+from .._parse import sklearn_operator_name_map
 import numpy as np
+from sklearn.preprocessing import OneHotEncoder
 
+class Oper:
```

Comment:
Why do you need this new class?

Reply:
Removed it. It was needed when I was using OHE converter.